src/util/common/hashmap.hpp

```diff
@@ -41,9 +33,7 @@ namespace cbdc::hashing {
     template<>
     struct const_sip_hash<buffer> {
         auto operator()(const buffer& buf) const noexcept -> size_t {
-            // TODO: pick the initialization key at random.
-            static constexpr std::array<uint64_t, 2> siphash_key{0x1337,
-                                                                 0x1337};
```

Comment:
Also curious whether this seems appropriate. Since the random values can't be determined at compile time, this update required making this value non-constant. Dynamic RNG seems like a good choice to me, but curious if static would be more appropriate here.